README.md

```diff
@@ -29,24 +29,45 @@ Fedora:
 sudo dnf install ocl-icd-devel
 ```
 
+Windows:
+- AMD GPU: [OCL-SDK](https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases/)
+- Nvidia GPU: [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit)
+
 ### Rust
 
+Linux:
+
 ```
 curl https://sh.rustup.rs -sSf | sh
 ```
 
-### Build and run
+Windows: follow instructions in https://www.rust-lang.org/tools/install
 
-```
+### Build
+
+```bash
 git clone https://github.com/nanocurrency/nano-work-server.git
 cd nano-work-server
+```
+
+Linux:
+
+```bash
 cargo build --release
-cd target/release
-./nano-work-server --help
 ```
 
+Windows:
+
+```bash
+cargo rustc --release
```

Comment:
This appears only be needed if linking otherwise `cargo build --release` will work, Maybe initially offer build --release and provide `rustc --release -- -l OpenCL -L "path/to/opencl.lib"` as a common issue callout?